web/src/cypress/integration/profile.ts

```diff
@@ -13,7 +14,12 @@ function countryCodeCheck(
     const name = 'CM SM ' + c.word({ length: 8 })
     const type = c.pickone(['SMS', 'VOICE'])
 
-    cy.pageFab('Contact')
+    if (screen === 'mobile') {
+      cy.pageFab('Add Contact Method')
```

Comment:
I considered having a second param to pageFab called "fallbackSelector" if a page fab isn't present. It just prioritizes mobile at that point. Thoughts?

Reply:
I think in this case doing the switch without the indirection is clearer.

The UI itself splits, and the test code reflects that. On wide, there's no FAB so calling `pageFab` would probably be confusing without context.

Reply:
Yeah that's pretty much where I landed. Sounds good 👍